base/compiler/ssair/inlining.jl

```diff
@@ -1120,13 +1119,14 @@ function process_simple!(ir::IRCode, todo::Vector{Pair{Int, Any}}, idx::Int, sta
         return nothing
     end
 
-    return sig
+    return stmt, sig
```

Comment:
this tuple looks very unfortunate

Reply:
I believe this tuple is okay since `sig` is a `Signature` object and so the tuple type is just concrete.